src/accel/TrackingManagerConstructor.cc

```diff
@@ -93,6 +101,7 @@ void TrackingManagerConstructor::ConstructProcess()
     auto* transporter = this->get_local_transporter();
     CELER_VALIDATE(transporter, << "invalid null local transporter");
 
+#if G4VERSION_NUMBER >= 1100
```

Comment:
If there is a validate that should prevent the construction of the class, why do we have this here?

Reply:
The code below it requires methods/types that won't compile in the newer versions.